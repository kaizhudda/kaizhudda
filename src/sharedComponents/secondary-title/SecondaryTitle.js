import React from 'react';

import './SecondaryTitle.scss';

const SecondaryTitle = ({ children }) => {
  return (
    <h3 className="SecondaryTitle col-xl-5 col-lg-6 col-md-9 col-xs-12">
      {children}
    </h3>
  );
}

export default SecondaryTitle;
