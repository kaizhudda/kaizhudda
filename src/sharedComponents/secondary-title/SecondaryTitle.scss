@import '../../styles/variables';
@import '../../styles/media-queries';

.SecondaryTitle {
  font-weight: 300;
  font-size: 1.75em;
  margin-top: 0px;
  color: #22a39f;
  text-align: left;
  width: 100%;
  margin-bottom: 15px;

  @include phone-only {
    font-size: 1.25rem;
  }

  @include phablet-only {
    font-size: 1.35rem;
  }
}