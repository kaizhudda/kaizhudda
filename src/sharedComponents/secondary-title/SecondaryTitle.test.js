import React from 'react';
import { shallow } from 'enzyme';
import SecondaryTitle from './index';

const secondaryTitle = 'Test Secondary Title';
let wrapped = shallow(<SecondaryTitle>{secondaryTitle}</SecondaryTitle>);

describe('SecondaryTitle', () => {
  it('should render the Secondary Title Component', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('should render the Secondary Titles title', () => {
    expect(wrapped.find('h3').text()).toEqual(secondaryTitle);
  });

});