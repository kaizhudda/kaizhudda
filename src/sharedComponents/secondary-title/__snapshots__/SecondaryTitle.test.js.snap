// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryTitle should render the Secondary Title Component 1`] = `
<h3
  className="SecondaryTitle col-xl-5 col-lg-6 col-md-9 col-xs-12"
>
  Test Secondary Title
</h3>
`;
